app/models/user/permissions.rb

```diff
@@ -39,27 +40,43 @@ def admin_sections=(sections)
     keep_permissions = existing_permissions.values_at(*keep)
     new_permissions = create.map { |section| member_permissions.build(admin_section: section) }
 
-    self.member_permissions = keep_permissions + new_permissions
+    self.member_permissions = keep_permissions + new_permissions + [services_member_permission].compact
   ensure
     @_admin_sections = nil
   end
 
-  def member_permission_service_ids=(service_ids)
-    services_section = [:services]
+  # returns all permissions (:portal, :finance, :settings, :partners, :monitoring, :plans) for admins
+  # and the allowed ones for member users
+  def member_permission_ids
+    admin? ? AdminSection.permissions : admin_sections - [:services]
```

Comment:
This method returns permissions for the current user, right? Name is a bit confusing because apparently there are _admin_ and _member_ users but the method name only mentions the latter. I think:
```ruby
def current_user_permission_ids
    admin? ? AdminSection.permissions : user_permissions
end

def user_permissions = admin_sections - [:services]
```
would be clearer. Also what's the difference between `AdminSection.permissions` and `admin_sections`?

Reply:
Changing the name is not simple at all in this case as Daria said 😄 However, she added an alias to access it with a clearer name:
https://github.com/3scale/porta/pull/121/files/fa9ac5a2fd1dd1a5fa5867577c775e176b7a5c46#diff-4fd20aa3226fcfc2bdf840fac716c74fR11

Reply:
![image](https://user-images.githubusercontent.com/11672286/49370069-66fad300-f6f3-11e8-8198-5e16c4175222.png)